md/connector-development-toolkit.md

```diff
@@ -316,3 +316,12 @@ The following example is the configuration for a Group widget that contains two
         </widgets>
 </widget>
 ```
+
+## Attention
+:warning:
```

Comment:
⚠️ probably not the right syntax

See https://github.com/bonitasoft/bonita-doc/blob/2707cb156b837e960bcb37bfdb8d07008a1889f4/md/api-extensions.md
```
::: warning
**Warning**: As the deployment of the REST API Extensions automatically creates the appropriate resource permission mapping, be careful not to [push a configuration](BonitaBPM_platform_setup.md#update_platform_conf) that does not contain the latest permissions updates. This would overwrite the deployed configuration. To avoid doing so, pull the latest configuration, edit the needed elements and push it back to the Bonita runtime in a pretty short amount of time.
:::
```